.github/actions/api-sdk-check/action.yml

```diff
@@ -20,10 +20,7 @@ runs:
     - name: Generate API SDK
       shell: bash
       run: |
-        set -ex
-        (cd apps/api && npm run generate-openapi)
-        (cd packages/api-sdk && speakeasy run --skip-versioning)
-        pnpm format
```

Comment:
_this_ was our root cause. every time we run the Code Quality check, we were formatting the repo running. the `format:check` script happened next, and it always passed because of this.